frontend/epfl-memento/templates/includes/card-info.inc.php

```diff
@@ -1,45 +1,65 @@
 <?php
 namespace EPFL\Plugins\Gutenberg\Memento;
 
-$start_date = date_format(date_create($event->start_date), 'd-m-Y');
-$end_date   = date_format(date_create($event->end_date), 'd-m-Y');
-$start_time = date_format(date_create($event->start_time), 'H:i');
-$end_time   = date_format(date_create($event->end_time), 'H:i' );
+$start_date = date_format(date_create($event->start_date), 'd.m.Y');
```

Comment:
This date format is Bulgarian (https://www.ibm.com/docs/fr/db2/11.5.x?topic=considerations-date-time-formats-by-territory-code). The Swiss one is `jj/mm/aaaa`, the same as UK.

Reply:
I'll still args in flavor of doing it the right way and then ask memento to do it right too.